docs/Pipelines/Stats_Surface.md

```diff
@@ -19,7 +19,6 @@ You need to process your data with the [`t1-freesurfer` pipeline](../T1_FreeSurf
 Do not hesitate to have a look at the paragraph **[Specifying what surface data to use](#advanced-specifying-what-surface-data-to-use)** if you want to use your own surface feature.
 
 ## Dependencies
-<!--If you installed the docker image of Clinica, nothing is required.-->
 
 If you only installed the core of Clinica, this pipeline needs the installation of **Matlab** and **FreeSurfer 6.0** on your computer.
```

Comment:
You didn't change this dependencies section like the others. Is it normal?

Reply:
Good catch ! 👍 
I didn't change it on purpose because the whole page is totally outdated.
We dropped SurfStat (to use Brainstat instead which is pure Python) more than one year ago but the documentation page was never updated.
There is an issue opened to remember us to do it one day: #980